src/Insurello.AsyncExtra/Insurello.AsyncExtra.fsproj

```diff
@@ -12,7 +12,12 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Update="FSharp.Core" Version="4.7.2" />
     <Compile Include="AsyncExtra.fs" />
   </ItemGroup>
+  <PropertyGroup>
+    <OtherFlags>$(OtherFlags) --warnon:1182 --warnaserror:FS0025</OtherFlags>
+    <!-- <TreatWarningsAsErrors>true</TreatWarningsAsErrors> -->

```

Comment:
Do we want this uncommented? If removed, should enforce this while building.

in the workflow use ` /WarnAsError` for the build command or `-warnaserror` in the publish command.

Reply:
In the publish to nuget?

Reply:
Yeah publish was is needed, build should be enough!